pkg/kube/inject/testdata/webhook/TestWebhookInject_mtls_not_ready.patch

```diff
@@ -21,7 +21,54 @@
       "prometheus.io/path": "/stats/prometheus",
       "prometheus.io/port": "15020",
       "prometheus.io/scrape": "true",
-      "sidecar.istio.io/status": "{\"initContainers\":null,\"containers\":null,\"volumes\":null,\"imagePullSecrets\":null}"
+      "sidecar.istio.io/status": "{\"initContainers\":[\"istio-init\"],\"containers\":[\"istio-proxy\"],\"volumes\":[\"istio-envoy\",\"istio-certs\"],\"imagePullSecrets\":[\"istio-image-pull-secrets\"]}"
```

Comment:
These are all changing because I accidentally merged the change to use the new format in a previous PR, so they were injection nothing.

I am starting to wonder if these tests are even useful...

Reply:
I wonder if we remove them what affect it would have on code coverage?